### PR TITLE
Fix BanchoBot article

### PR DESCRIPTION
**Pull request by Shadowtail117**
- Created a table of all of the standard commands.
- Created a table for all of the `/bb` commands.
- Added some in-line "trivia" (before each table). 

(Note: The names used are arbitrary)

### CODE REVIEW

File: wiki/BanchoBot/en.md [hunk 1]
```diff
@@ -2,36 +2,48 @@
...
+| `/nopm` | N/A | Quickly toggle whether you want to receive private messages from friends or not. | /nopm | (Displayed as a banner in the center of the screen) "You are now ignoring all private messages from everyone except your friends." When toggled again, "You are not longer ignoring pivate messages from non-friends." |
+| `/invite` | `<user>` |  Sends a multiplayer invitation to the specified user with a link to the room *(You must be in a match to invite someone.)* | /invite PikaFan025 | (Displayed as a banner in the center of the screen) "PikaFan025 has been invited to your game." On the recipient's end, a chat tab with you opens (if it hasn't already) and the text "\<username\>: Come join my multiplayer match: "[\<Room name\>](/wiki/BanchoBot/en.md)" |
+
+### `/bb` Commands
```
Comment (Shadowtail117) [inline wiki/BanchoBot/en.md]: Note: I couldn't figure out how `SEARCH` worked, so I omitted it. If anyone else knows, feel free to comment.

**Comment by Shadowtail117** [comment]
You can view the file on my fork; that way the removed columns aren't visible and it's actually readable :P

File: wiki/BanchoBot/en.md [hunk 2]
```diff
@@ -2,36 +2,48 @@
...
-
-## BanchoBot Quirks
-
-- Commands are case insensitive.
-- If you are not on the BanchoBot tab, you must use the `!` or `/bb` prefix.
-- If you have already called BanchoBot (with tab available), any command you use will still appear at the BanchoBot tab regardless of the tab you are currently in when you issue the command.
-- You can enter commands to BanchoBot without the `!` or `/bb` prefix (only works when submitted to BanchBot tab).
+**Note: You *must* prefix all commands with a `/`!** All effect listings are verbatim from the command list. Notes are shown in *italics*.
+
+### Standard Commands
```
Comment (MegaApplePi) [inline wiki/BanchoBot/en.md]: remove this section #313 will take care of the client side commands

if you think it is appropriate, you could instead replace this section with:
```
_See also [Chat Console](/wiki/Chat_Console)._
```

File: wiki/BanchoBot/en.md [hunk 3]
```diff
@@ -2,36 +2,30 @@
...
-| `FAQ <Lang-code><item> <list>` | Various usage. [list] List of usable commands.                               | `!faq wiki` / `!faq ru:wiki`        | The osu! Wiki - Make it awesome! / Примите участие в заполнении вики!   |
-| `REPORT <reason>`              | Call for a moderator                                                         | `!report Thomas offending comments` | Chat moderators has been alerted. Thanks for your help.                 |
-| `REQUEST <list>`               | Show a random recent mod request. [list] Shows 5 random recent mod requests. | `/bb request`                       | Seether - Fake It by [Dellirium]                                        |
-| `ROLL <number>`                | Roll a (virtual) dice and get random result from 1 to (number) (default 100) | `/bb roll 9000`                     | John rolls 1337 point(s)                                                |
+### Standard Commands
 
-Notes:
+For a table of standard commands, go to the [Chat Console](/wiki/Chat_Console) article.
```
Comment (MegaApplePi) [inline wiki/BanchoBot/en.md]: italics?

**Pull request merged**